lib/variables.c

```diff
@@ -51,76 +51,74 @@ variable_create_esl(void *cert, int cert_len, EFI_GUID *type, EFI_GUID *owner,
 	return EFI_SUCCESS;
 }
 
-
```

Comment:
These changes are not GUID-related, but whitespace. Looks like they belonged to some previous commit.

Reply:
Thanks, fixed in my next version.